deploy/backplane/elevated-sre/20-cluster-admin.User.yml

```
@@ -1,4 +1,5 @@
 apiVersion: user.openshift.io/v1
 kind: User
 metadata:
-  name: backplane-cluster-admin
\ No newline at end of file
+  name: backplane-cluster-admin
+groups: []
```

```suggestion
groups: null
```
When a User is created w/o `groups` the value defaults to `null`, suggest we keep consistency with that default.